mwdb/web/src/components/AttributesAddModal.js

```diff
@@ -35,6 +38,16 @@ export default function AttributesAddModal({ isOpen, onAdd, onRequestClose }) {
 
     function handleKeyChange(ev) {
         setAttributeKey(ev.target.value);
+        if (!ev.target.value.length) setRichTemplate("");
+        else {
+            setRichTemplate(
+                attributeDefinitions[ev.target.value].rich_template
+            );
+            setAttributeValue(
+                attributeDefinitions[ev.target.value].example_value || ""
+            );
+        }
```

Comment:
As example value represents JSON-stringified object, target type should be object
```suggestion
        }
        setAttributeType("object");
```